custom/icds_reports/static/js/directives/indie-map/indie-map.directive.js

```diff
@@ -163,6 +163,20 @@ function IndieMapController($scope, $compile, $location, $filter, storageService
                     projection.scale(scaleCenter.scale)
                         .center(scaleCenter.center)
                         .translate([element.offsetWidth / 2, element.offsetHeight / div]);
+
+                    //setting zoom out limit
+                    //references: https://data-map-d3.readthedocs.io/en/latest/steps/step_06.html#step-06
+                    $(function(){
+                        // DOM Ready
+                        var svg = d3.select('#map svg'); //selects svg in datamap component
+                        var zoom = d3.behavior.zoom().scaleExtent([1, 10]).on('zoom', function () {
+                            // this function redraws the map rendered on zoom event
+                            // reference: bower_components/angular-datamaps/dist/angular-datamaps.js (line 27)
+                            svg.selectAll('g').attr("transform",
+                              "translate(" + d3.event.translate + ") scale(" + d3.event.scale + ")");
```

Comment:
Sorry if I'm being dumb, but I still don't understand how this works. What is the line or value that determines the maximum level of zoom out? Note that this value would vary based on what geography you are viewing.

Reply:
This part doesn't decide zoom level. It just redraws the map based on zoom level. zoom level is decided by scale extent. (mentioned in the above comment)

Reply:
Are you saying max zoom level would vary based on geo

Reply:
Yes, when you're zoomed into a district, for example the max zoom should be higher since the geography is much smaller.

Reply:
(offlined, this works as expected)